src/Symfony/Component/Validator/ConstraintValidatorFactory.php

```diff
@@ -42,7 +39,7 @@ public function getInstance(Constraint $constraint)
 
         if (!isset($this->validators[$className])) {
             $this->validators[$className] = 'validator.expression' === $className
```

Comment:
well, the validator still has a dependency on expression language which could be injected as constructor argument. @fabpot do we not have a tag to add expression function providers to the expression language? this seems to be missing for the validator here. I guess it only support the default expression language so far.